.github/workflows/ci-cleanup-old-workflow-runs.yml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
missing license header

Reply:
The headers are missing in all workflow files. To track the effort of fixing this everywhere, I created https://github.com/magma/magma/issues/13280.